From 1e9449c06693c07c30b47f9c9589b99aae8d5479 Mon Sep 17 00:00:00 2001
From: Dan Poltawski <dan@moodle.com>
Date: Fri, 29 Jul 2016 12:54:22 +0100
Subject: [PATCH 1/1] MDL-12345 summary: the first line is good!

But then it all goes wrong, with a message that goes really quite far down to the depths of the right hand side of the screen. This person has never heard of wrapping text, sad eh!
---
 README.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/README.txt b/README.txt
index 729dbe4..af81703 100644
--- a/README.txt
+++ b/README.txt
@@ -26,3 +26,5 @@ Moodle is written in PHP and JavaScript and uses an SQL database for storing
 the data.
 
 See <https://docs.moodle.org> for details of Moodle's many features.
+
+a dummy commit!
-- 
2.9.0

